app/src/main/res/drawable/nps_survey_button_inactive_background.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <corners android:radius="@dimen/state_previous_next_button_radius" />
```

Comment:
Check the dimensions file for a survey screen-specific radius, or create one.

Reply:
Done.